minutes/board_2020_11_12.md

```diff
@@ -0,0 +1,89 @@
+    * Setup January meeting to discuss NiT goals
+         * In progress
+* Nick
+    * Look into setting up vpn for NiT
+        * See Updates below
+* Coty
+    * Create a Google Form to get emails from members interested in joining the membership list and send out in #general.
+        * No progress, basically done no work since the new baby while I’m off work. Committed to doing this before the next meeting.
+
+### Updates
+
+* Adam
+    * Conference
+        * Conference is going take place Nov 21st
+        * Details: https://blog.nativesintech.org/natives-in-tech-conf-2020/
+        * Got a lot of help from Ana Hobden, probably wouldn’t have happened without her
+        * We also got some sponsorship money from Asquera and Hoverbear Consulting!
+    * Blog
+        * Blog can now send out newsletters, thanks nick@nativesintech.org
+    * Slack
+        * Invites are invite only because of bots
+    * Coding Dojo
+        * Update: 5 free scholarships for Dec!
+    * Documentation
+        * Docs for NiT and NiT, Inc can be found at https://docs.nativesintech.org/
+        * They are a work in progress
+        * My plan is to write down _everything_ we do that drives NiT: buffer.com for social media, netlify for deployments, digital ocean for hosting, azlo for banking, open collective for accepting donations,  etc.
+    * Rebrand
+        * Rebrand of all the things: website, facebook, forum, linkedin, etc.
+    * Talent
+        * NiT Talent is live! People can make recommendations
+    * LinkedIn
+        * Made everyone here an admin on our NiT site on LinkedIn
+    * 1Password
+        * Nick, Erin, and Coty are now team members
+        * ~~Waiting on Coty :)~~
+        * Nick added all mailgun info to 1password
+            * I updated mailgun to have 2FA w/1password instead of email
+        * Try to use 1Password for 2FA as well as passwords
+        * Need to save 1 time password on machine or somewhere like keybase
+    * January Meeting
+        * I’ve written out the State of NiT and will share to get peoples feedback
+        * I will send out ideas next month
+* Nick
+    * Started work on native-owned businesses again 😬
+        * Started building create-business components and page
+    * Haven’t tried VPN experiment yet but will this week!
+    * Updated discourse forums
+        * Latest version
+        * Also updated discourse theme to match branding
+* Coty
+    * Everyone please review PRs in the nonprofit-governance repo, including minutes from previous meetings
+    * Slowly working on an online course titled “How the Internet Works'' for beginners. Idea is to try and give users basic working knowledge of the net and give them useful information about how to stay safe now that everyone is online these days.
+* General
+    * Paying annual dues
+
+### Meetings Notes
+
+### Action Items
+
+* Adam
+    * Review NiT Governance PRs
+    * Put backup codes for all 1password logins on keybase
```

Comment:
Agenda item for next meeting: Are we still using Keybase since it got bought out?

Reply:
Yes, for back up codes.